test/test-test.js

```diff
@@ -67,26 +71,20 @@ describe('svg-react-loader', function () {
                     throw error;
                 }
 
-                var src = babel.transform(result, {
-                    presets: ['es2015', 'react']
-                }).code;
-                console.log(src);
-                fs.writeFileSync(__dirname + '/temp', src);
-                var el = react.createElement(require(__dirname + '/temp'));
-                var html = react.renderToStaticMarkup(el);
```

Comment:
I got problems reading the file back in. It seems as if the previous file was read back in.. But I don't think you need to write to disk an read back in here. An `eval()` will also work fine.